src/Numeric/Sundials.hs

```diff
@@ -258,6 +259,23 @@ withCConsts ODEOpts{..} OdeProblem{..} = runContT $ do
       DenseJacobian -> 0
     c_method = methodToInt odeMethod
 
+  exceptionRef <- ContT $ bracket newEmptyMVar $ \mvar -> do
+    -- If an exception happened during the execution of the solver it at
```

Comment:
there is a problem with this sentence, looks like it lacks a verb maybe?

Reply:
```suggestion
    -- If an exception happened during the execution of the solver in a
```

Reply:
Yes, you are right. Fixed in suggestion batch.